lib/diagnostic.rb

```diff
@@ -33,8 +43,9 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
-Response.dave = dave
+dave = Person.new("Dave", 32, "Ohio")
+  Person[:location] << "Somerville"
+Response.dave = Person.new
 ##
```

Comment:
I think you're confusing some syntaxes here, today's SQL talk should clear things up.